plugins/pulp_rpm/plugins/importers/yum/sync.py

```diff
@@ -1040,6 +1069,9 @@ def _identify_wanted_versions(self, package_info_generator):
             for unit, info in units.itervalues():
                 ret[unit] = info
 
+        if package_type == ids.TYPE_ID_RPM:
```

Comment:
Yes, that makes sense. Thanks for putting all the thoughts, ideas and explanations, that's always helpful!